Character.py

```diff
@@ -1,18 +1,54 @@
 import random
+import json
 
 class Character:
     def __init__(self):
-        self.Name = random.choice(['Kevin', 'Alice', 'Dri', 'J'])
+        #self.Name = random.choice(['Kevin', 'Alice', 'Dri', 'J'])
         pass
 
+
+    def isProficient(self, skill):
+        for k in self.Proficiencies:
+            if k == skill:
+                return True
+        return 
+    
+    def isExpert(self, skill):
+        for k in self.Expertise:
+            if k == skill:
+                return True
+        return
+
+    
+
     @classmethod
-    def fromJson(cls, jsonObj):
+    def fromJson(cls):
```

Comment:
This should take some kind of parameter to specify which character to load.

Reply:
>             This should take some kind of parameter to specify which character to load.

Like (self, skill, name)?